code/datums/components/caltrop.dm

```diff
@@ -59,4 +61,7 @@
 						"<span class='userdanger'>You slide on [A]!</span>")
 
 			cooldown = world.time
-		H.Paralyze(60)
+		if(is_species(H, /datum/species/squid))
+			H.Paralyze(5)
```

Comment:
These values are still stupid-low. Make it 10 for squids and 40 for everything else and I'll approve this PR.

Reply:
Fuck yes